source/usage-examples/insertOne.txt

```diff
@@ -11,6 +11,7 @@ Insert a Document
    :values: tutorial
 
 .. meta:: 
+   :description: Learn how to insert a document into your MongoDB collection by using the Node.js driver.
```

Comment:
Suggestion:

Similar to the suggestions provided for source/usage-examples/find.txt, I think one of these would keep them similar to each other:

```suggestion
   :description: Learn how to insert a document into MongoDB by using the Node.js driver.
```

```suggestion
   :description: Follow a usage example of inserting a document into MongoDB by using the Node.js driver.
```